src/watchmaker/utils/__init__.py

```diff
@@ -83,3 +84,31 @@ def copytree(src, dst, force=False, **kwargs):
         shutil.rmtree(dst)
 
     shutil.copytree(src, dst, **kwargs)
+
+
+def config_none_deprecate(check_value, log):
+    r"""
+    Check whether a variable is the string 'None' rather than Pythonic `None`.
+
+    If it is the string 'None', this warns and returns Pythonic `None`.
+    Otherwise, the variable is returned unchanged.
+
+    Args:
+        check_value: (:obj:`str`)
+            Variable to be checked for string 'None' (case-insenstive).
+            (*Default*: None)
+
+        log: (:obj:`logging.Logger`)
+            Logger where deprecation warning will be made.
+            (*Default*: None)
+
+    """
+    if check_value and check_value.lower() == 'none':
+        deprecate_msg = (
+            'Use of "None" (string) as a config value is deprecated. Use '
+            '`null` instead.')
+        log.warn(deprecate_msg)
+        warnings.warn(deprecate_msg, DeprecationWarning)
```

Comment:
Do we get two log outputs when we do both log.warn and warnings.warn?

Reply:
No. `warnings.warn`'s behavior can be customized to do things like raising an error, filter out warnings, or possibly log the error. The default is to output to stderr. Figured this would be another way to gradually ramp up the deprecation consequences.